resources/testing_logger.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<logging>
+    <loggerOptions>
+        <loggerOption LoggerName="FileLogger"   >true</loggerOption>
+        <loggerOption LoggerName="ConsoleLogger">false</loggerOption>
+    </loggerOptions>
+    <logLevels>
+        <logLevel level="ERROR"  >true</logLevel>
+        <logLevel level="WARNING">false</logLevel>
+        <logLevel level="SUCCESS">false</logLevel>
+        <logLevel level="INFO"   >false</logLevel>
```

Comment:
Completely slipped my mind. In testing what we do not want is the module name and the timestamp.

Reply:
Also ERROR, SUCCESS and INFO can be true

Reply:
> In testing what we do not want is the module name and the timestamp

could have told it a lot sooner because that is indeed tricky
given the current setup, the XML file is being parsed by the LoggerManager 
while the module name and timestamps are decided by the consoleLogger and fileLogger. 

If you want to disable those, then the only way out that I see is:
to have more arguments to the constructors which will then bring testMode 'state' to the independent loggers.
then have the module and timestamps be kept unused (sidenote: it was a good decision that I did those separately before creating the final message format :`D)
let me know if there is a different way

Reply:
Can you make error, success and info true as well

Reply:
okay